packages/apollo-server-core/src/index.ts

```diff
@@ -41,6 +41,18 @@ export const gql: (
   ...substitutions: any[]
 ) => DocumentNode = gqlTag;
 
+import supportsUploadsInNode from './utils/supportsUploadsInNode';
 import { GraphQLScalarType } from 'graphql';
-import { GraphQLUpload as UploadScalar } from '@apollographql/apollo-upload-server';
-export const GraphQLUpload = UploadScalar as GraphQLScalarType;
+export { default as processFileUploads } from './processFileUploads';
+
+// This is a conditional export intended to avoid traversing the
+// entire module tree of `graphql-upload`.  This only defined if the
+// version of Node.js is >= 8.5.0 since those are the only Node.js versions
+// which are supported by `graphql-upload@8`.  Since the source of
+// `graphql-upload` is not transpiled for older targets (in fact, it includes
+// experimental ECMAScript modules), this conditional export is necessary
```

Comment:
`graphql-upload` supports both CJS, *and* native ESM (if the consumers Node.js is running with the `--experimental-modules` flag). Both the CJS and ESM builds are transpilled, the only difference is the module format. But yes, there is a chance some new API or syntax might throw older Node.js if imported.